js/client-chat.js

```diff
@@ -560,6 +560,16 @@
 				}
 				return false;
 
+			case 'news':
+				var newsId = '1990';
+				app.rooms[''].addPseudoPM({
+					title: 'News',
+					html: '<iframe src="/news-embed.php?news' + (window.nodewebkit || document.location.protocol === 'https:' ? '&amp;https' : '') + '" width="270" height="400" border="0" style="border:0;width:100%;height:100%;display:block"></iframe>',
```

Comment:
How is News currently implemented? We should probably piggy-back off the existing implementation, rather than creating a new imitation one.

Reply:
news is currently implemented in mainmenu as 
`		addNews: function () {
			var newsId = '1990';
			if (newsId === '' + Dex.prefs('readnews')) return;
			this.addPseudoPM({
				title: 'Latest News',
				html: '<iframe src="/news-embed.php?news' + (window.nodewebkit || document.location.protocol === 'https:' ? '&amp;https' : '') + '" width="270" height="400" border="0" style="border:0;width:100%;height:100%;display:block"></iframe>',
				attributes: 'data-newsid="' + newsId + '"',
				cssClass: 'news-embed',
				height: 400
			});
		},`

Reply:
and added within the loading process

Reply:
Actually, i can just call it directly it seems.